invenio_rdm_records/records/api.py

```diff
@@ -57,7 +56,9 @@ class CommonFieldsMixin:
 
     access = AccessField()
 
-    status = RecordStatusField()
+    # We redefine the property as we extend the `PIDStatusCheckField` to dump
+    # the property in ES in order to be available for aggregation
+    is_published = IsPublishedField(status=PIDStatus.REGISTERED)
```

Comment:
Comment: The name should be aligned with the existing name in the service schema - not sure if it is or not - but from top of my head I think it's named ``published``. Just check and if different we have to update one of the two.